packages/config-utils/proxy.js

```diff
@@ -252,6 +254,8 @@ module.exports = ({
                     });
                 }
 
+                onBeforeSetupMiddleware({ chromePath });
```

Comment:
can be chromePath null? there is a condition to check it just below it

Reply:
I think I will leave it in here because I think we should have a consistent API, but add a check in the consumer.